iam-login-service/src/main/java/it/infn/mw/iam/config/MitreServicesConfig.java

```diff
@@ -289,5 +290,9 @@ DeviceCodeService defaultDeviceCodeService() {
   SelfAssertionValidator selfAssertionValidator() {
     return new SelfAssertionValidator();
   }
-  
+
+  @Bean
+  IDTokenClaimsEnhancer defaultIdTokenClaimsEnhancer() {
```

Comment:
Use the Service or Component annotation in the class to declare the component (see below)